src/core/Akka/Serialization/Serialization.cs

```diff
@@ -210,17 +205,24 @@ public Serialization(ExtendedActorSystem system)
                     continue;
                 }
 
-
                 if (!_serializersByName.TryGetValue(serializerName, out var serializer))
                 {
                     system.Log.Warning("Serialization binding to non existing serializer: '{0}'", serializerName);
                     continue;
                 }
-
                 AddSerializationMap(messageType, serializer);
             }
 
-            
+            // Add any serializer bindings that are registered via the SerializationSetup
+            // This has to be done here because SerializationSetup ALWAYS win.
+            foreach (var details in _serializerDetails)
```

Comment:
yep -- I suspected it was an ordering issue -- applying the defaults after apply the user defined settings. easy peasy!